sklearn/tree/tests/test_tree.py

```diff
@@ -2826,7 +2826,7 @@ def test_sort_log2_build():
     rng = np.random.default_rng(75)
     some = rng.normal(loc=0.0, scale=10.0, size=10).astype(np.float32)
     feature_values = np.concatenate([some] * 5)
-    samples = np.arange(50)
+    samples = np.arange(50, dtype=np.float64)
```

Comment:
```suggestion
    samples = np.arange(50, dtype=np.intp)
```

Reply:
No sweat! I force-pushed the fix :) thank you!